relay/metadata_server.go

```diff
@@ -0,0 +1,153 @@
+package relay
```

Comment:
Why do we need relay to serve blob metadata? 
I think the only service it needs are blobs & chunks delivery

Reply:
This is for internal relay use. When an external user requests data, the relay needs to first fetch the metadata. It uses the metadata to decide if the blob is in the correct relay shard, as well as to figure out the proper file sizes for downloading fragmented files.

Reply:
Ah I should have taken a closer look. This is a library for fetching the metadata, not a server 😭 
Can we rename it to something other than "server"

Reply:
I've renamed these helper functions to `blobManager`, `chunkManager`, and `metadataManager`.